packages/core/src/modules/slack/use-cases/update-birthdates-to-db.ts

```diff
@@ -0,0 +1,41 @@
+import { db } from '@/infrastructure/database';
+import { slack } from '@/modules/slack/instances';
+import { RateLimiter } from '@/shared/utils/rate-limiter';
+
+const getBirthdatesRateLimiter = new RateLimiter(
+  'slack:connections:get_birthdates',
+  {
+    rateLimit: 100,
```

Comment:
The method `users.profile.get` has a [Tier 4 Rate Limit](https://api.slack.com/methods/users.profile.get) so the limit is 100+/minute